soot-infoflow-collections/src/soot/jimple/infoflow/collections/CollectionsSetupApplication.java

```diff
@@ -0,0 +1,80 @@
+package soot.jimple.infoflow.collections;
```

Comment:
Hier eine neue Klasse einzuführen ist heikel. Mit den Änderungen an den Summaries macht FlowDroid mit der "normalen" Klasse eine unter-Approximation, weil Hinzufügungen zur Collections nun gar nicht mehr behandelt werden, also die Collection gar nicht tainted wird. Die neuen Klassen kennt niemand. Wenn jemand also einfach sein FlowDroid aktualisiert, hat er False Negatives. Das müssen wir anders machen. Schlimmstenfalls benennen wir die alte Klasse um oder sowas, damit den Leuten zumindest auffällt, dass sich etwas geändert hat. Generell haben wir aber erheblich Copy&Paste-Code, was bzgl. Maintenance nicht ideal ist.

Reply:
Das versteh ich nicht ganz. Wenn ich kein Blödsinn gebaut habe, sollte sich an FlowDroid nichts geändert haben. Ich hab die Klasse nur erstellt, damit ich nicht überall inline den RuleManager und TaintWrapper ersetzen muss. IMO kann man die CollectionsSetupApplication bzw. CollectionsInfoflow einfach löschen. Dann muss man nur kurz das halt in VUSC stattdessen anpassen.